appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
Definitely agree we don't want to call the GitHub API, would much prefer using `git describe`, however the AppVeyor environment variable seems to work good.

Can we fix AppVeyor setting the version to `0.5.276` when the tag is `0.5.6`, it also seems wrong having `version: 0.5.{build}` for all builds on any branch?